tests/smrf_test_case.py

```diff
@@ -101,10 +117,47 @@ def setUpClass(cls):
         # read in the base configuration
         cls.base_config = get_user_config(config_file, modules='smrf')
 
-    def tearDown(self):
+    @classmethod
+    def tearDownClass(cls):
+        """
+        Clean up the output directory
+        """
+        folder = os.path.join(cls.base_config.cfg['output']['out_location'])
+        if os.path.exists(folder):
+            shutil.rmtree(folder)
+
+
+class SMRFTestCaseLakes(SMRFTestCase):
```

Comment:
I would vote to put this in it's own file

Reply:
new file